components/cache.rst

```diff
@@ -70,6 +70,14 @@ Now you can create, retrieve, updated and delete items using this cache pool::
 
     // remove the cache item
     $cache->deleteItem('stats.num_products');
+    
+Or you can use redis-based cache, instantiate :class:`Symfony\\Component\\Cache\\Adapter\\RedisAdapter`::
```

Comment:
I'd prefer to not add this Redis example here. Why? Because in the "Basic usage" part of the components we display a "Hello World" example ... something very simple to give a glimpse of the features of that component. This example should be moved to the other articles that explain each of the cache types available.